newIDE/app/src/CommandPalette/CommandPalette/AutocompletePicker.js

```diff
@@ -142,6 +144,7 @@ const AutocompletePicker = (
 
   const getItemHint = React.useCallback(
     (item: Item) => {
+      if (windowWidth === 'small' || windowWidth === 'medium') return null;
```

Comment:
are we sure this doesn't affect other components using AutocompletePicker with a hint?

Reply:
AutocompletePicker is used by CommandPalette only